src/Sulu/Bundle/PreviewBundle/Tests/Unit/Preview/Object/PreviewObjectProviderRegistryTest.php

```diff
@@ -15,54 +15,47 @@
 use Sulu\Bundle\PreviewBundle\Preview\Exception\ProviderNotFoundException;
 use Sulu\Bundle\PreviewBundle\Preview\Object\PreviewObjectProviderInterface;
 use Sulu\Bundle\PreviewBundle\Preview\Object\PreviewObjectProviderRegistry;
-use Sulu\Bundle\PreviewBundle\Preview\Object\PreviewObjectProviderRegistryInterface;
 
 class PreviewObjectProviderRegistryTest extends TestCase
 {
-    /**
-     * @var PreviewObjectProviderRegistryInterface
-     */
-    private $objectProviderRegistry;
-
-    /**
-     * @var PreviewObjectProviderInterface
-     */
-    private $provider;
-
-    /**
-     * @var string
-     */
-    private $providerKey = 'test-provider';
-
-    public function setUp(): void
-    {
-        $this->provider = $this->prophesize(PreviewObjectProviderInterface::class);
-
-        $providers = [$this->providerKey => $this->provider->reveal()];
-
-        $this->objectProviderRegistry = new PreviewObjectProviderRegistry($providers);
-    }
-
     public function testGetPreviewObjectProviders(): void
     {
-        $objectProviders = $this->objectProviderRegistry->getPreviewObjectProviders();
+        $provider = $this->prophesize(PreviewObjectProviderInterface::class);
+        $providerKey = 'test-provider';
+        $providers = [$providerKey => $provider->reveal()];
+        $previewObjectProviderRegistry = new PreviewObjectProviderRegistry($providers);
+
+        $previewObjectProviders = $previewObjectProviderRegistry->getPreviewObjectProviders();
 
-        $this->assertCount(1, $objectProviders);
-        $this->assertArrayHasKey($this->providerKey, $objectProviders);
-        $this->assertArrayNotHasKey('wrong-key', $objectProviders);
+        $this->assertCount(1, $previewObjectProviders);
+        $this->assertArrayHasKey($providerKey, $previewObjectProviders);
+        $this->assertArrayNotHasKey('wrong-key', $previewObjectProviders);
     }
 
     public function testGetPreviewObjectProvider(): void
     {
-        $this->assertEquals($this->provider->reveal(), $this->objectProviderRegistry->getPreviewObjectProvider($this->providerKey));
+        $provider = $this->prophesize(PreviewObjectProviderInterface::class);
+        $providerKey = 'test-provider';
+        $providers = [$providerKey => $provider->reveal()];
+        $previewObjectProviderRegistry = new PreviewObjectProviderRegistry($providers);
+
+        $this->assertEquals(
+            $provider->reveal(),
+            $previewObjectProviderRegistry->getPreviewObjectProvider($providerKey)
+        );
 
         $this->expectException(ProviderNotFoundException::class);
```

Comment:
Actually I meant that this should be two different tests... If we split tests like that, it will let you know what is wrong faster, because the result already says if it failed when asking for a non-existing provider or when returning an existing one.